### ISSUE TITLE
Magic numbers 

### ISSUE THREAD

**Issue by ArtificialEdge** [dichotomies/proxy-nca#12]
Apart from trial and error, is there any particular reason for using a multiplicative factor of 3 in the loss function. Is this the same reason why gradients are clipped above the value 10

**Comment by dichotomies** [comment]
I've selected the gradient clipping value through trial and error. That said, I remember that gradient clipping boosted the training stability and accuracy while I experimented with some implementation details, but I didn't double check if gradient clipping actually benefits the training for the final (this) implementation. I've chosen 3 as multiplicative factor with a reason.
